### PR TITLE
Update Firestore Changelog for 10.3.0

### PR DESCRIPTION
**Pull request by ehsannas**

### CODE REVIEW

**Review by andrewheard — APPROVED** [review]
Thanks!

File: Firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
-# Unreleased
-- [fixed] Fix App crashed when there are nested data structures inside IN Filter.
-
 # 10.3.0
 - [feature] Add MultiDb support.
+- [fixed] Fix App crashed when there are nested data structures inside IN
+  Filter (#10535).
```
Comment (paulb777) [inline Firestore/CHANGELOG.md:4]: Should be the issue number when there is both a PR and issue - #10507

Comment (andrewheard) [reply Firestore/CHANGELOG.md]: @ehsannas I've changed this to the issue number and committed to your PR.

Comment (ehsannas) [reply Firestore/CHANGELOG.md]: Oh! TIL. Thanks!

**Review by paulb777 — APPROVED** [review]
Thanks!

**Pull request merged**